Source/ACE.Server/Entity/DamageEvent.cs

```diff
@@ -423,6 +424,7 @@ public void ShowInfo(Player player)
             {
                 info += $"ElementalDamageBonus: {BaseDamageMod.ElementalBonus}\n";
                 info += $"MissileWeaponModifier: {BaseDamageMod.DamageMod}\n";
+                info += $"BloodDrinker/ThirstTotal: {BaseDamageMod.DamageBonus}\n";
```

Comment:
possible suggestion to keep the # of lines in DebugDamage in check:

add bonus to BaseDamageRange display as (#) on the end?

```
            // base damage
            if (BaseDamageMod != null)
            {
                var damageBonus = BaseDamageMod.DamageBonus;
                var damageBonusStr = "";

                if (damageBonus != 0)
                {
                    var plusMinus = damageBonus > 0 ? "+" : "";
                    damageBonusStr = $" ({plusMinus}{damageBonus})";
                }

                info += $"BaseDamageRange: {BaseDamageMod.Range}{damageBonusStr}\n";
            }
```

Reply:
I could put Elemental,Missile Modifier and DamageBonus on 1 line

Reply:
yeah that would be good too